osu.Game/Overlays/Settings/SettingsButton.cs

```diff
@@ -1,17 +1,33 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System.Collections.Generic;
+using System.Linq;
 using osu.Framework.Graphics;
+using osu.Framework.Graphics.Cursor;
 using osu.Game.Graphics.UserInterface;
 
 namespace osu.Game.Overlays.Settings
 {
-    public class SettingsButton : TriangleButton
+    public class SettingsButton : TriangleButton, IHasTooltip
     {
         public SettingsButton()
         {
             RelativeSizeAxes = Axes.X;
             Padding = new MarginPadding { Left = SettingsOverlay.CONTENT_MARGINS, Right = SettingsOverlay.CONTENT_MARGINS };
         }
+
+        public string TooltipText { get; set; }
+
+        public override IEnumerable<string> FilterTerms
+        {
+            get
+            {
+                if (TooltipText != null)
+                    return base.FilterTerms.Append(TooltipText);
```

Comment:
Why is this only done for SettingsButton?

Reply:
That's the only place we use tooltips right now.

Reply:
Only because they were implemented in this PR. Would rather all this code was moved to `SettingsItem`.

Reply:
Turns out this class is not even a `SettingsItem`. 🤦‍♂️